pkg/api/v1/schema.go

```diff
@@ -32,6 +32,22 @@ const (
 	IOThreadsPolicyAuto   IOThreadsPolicy = "auto"
 )
 
+// Represents the firmware blob used to assist in the domain creation process.
+// Used by Xen fully virtualized domains as well as setting the QEMU BIOS file path for QEMU/KVM domains
```

Comment:
we don't support Xen. Lets just document qemu/kvm

Reply:
Sounds good.